src/main/java/seedu/address/model/food/Food.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.model.food;
+
+import java.util.Objects;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
```

Comment:
I think you can import static class first then import java class, this should solve the checkstyle issue.

Reply:
Oh yes, thanks Shee Xiong! I ran my check style and it told me the same thing too